inductiva/tests/simulators/test_dummy_simulator.py

```diff
@@ -0,0 +1,89 @@
+"""Tests on a dummy simulator for testing purposes."""
+from typing import Optional, List
+import tempfile
+import json
+import os
+
+from inductiva import simulators, tasks, types
+
+INPUT_DIR = os.path.join(os.path.dirname(__file__), "test_input_dir")
+BACKEND_ARGS_FILE = "test_arguments.json"
+TEMP_DIR = tempfile.TemporaryDirectory()
+
+class DummySimulator(simulators.Simulator):
+    """Dummy simulator for testing purposes."""
+
+    def __init__(self):
+        super().__init__()
+        self.api_method_name = "tester.tester.run_simulation"
```

Comment:
No, we don't. In fact, this is something that is bothering for the general simulators, since we have the first name also referring to the type of numerical method/area/equations in use, which is not very structured. In principle, the middle name is enough: openfoam, gromacs, etc... would now be enough to call which method to use. See this issue: inductiva/tasks#47.